src/main/java/com/commercetools/sync/inventory/utils/InventoryDraftTransformer.java

```diff
@@ -0,0 +1,76 @@
+package com.commercetools.sync.inventory.utils;
+
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+import io.sphere.sdk.inventory.InventoryEntryDraftBuilder;
+import io.sphere.sdk.types.CustomFields;
+import io.sphere.sdk.types.CustomFieldsDraft;
+import io.sphere.sdk.types.Type;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+//TODO delete this when SDK release their transformer.
+/**
+ * This class provides static utility methods for converting objects of {@link InventoryEntry}
+ * into {@link InventoryEntryDraft}.
+ */
+public final class InventoryDraftTransformer {
+
+    private InventoryDraftTransformer() {
+        throw new AssertionError();
+    }
+
+    /**
+     * Returns new {@link InventoryEntryDraft} containing same data as {@code inventoryEntry}, including
+     * {@code supplyChannel} and {@code customFields} (which will be converted to {@link CustomFieldsDraft}).
+     *
+     * @param inventoryEntry {@link InventoryEntry} from which draft will be created
+     * @return {@link InventoryEntryDraft} created for passed {@code inventoryEntry}
+     */
+    public static InventoryEntryDraft transformToDraft(@Nonnull final InventoryEntry inventoryEntry) {
+        return InventoryEntryDraftBuilder
+                .of(inventoryEntry.getSku(),
+                    inventoryEntry.getQuantityOnStock(),
+                    inventoryEntry.getExpectedDelivery(),
+                    inventoryEntry.getRestockableInDays(),
+                    inventoryEntry.getSupplyChannel())
+                .custom(getCustomFieldsDraft(inventoryEntry.getCustom()))
+                .build();
+    }
+
+    /**
+     * Returns new {@link List} of {@link InventoryEntryDraft} created from {@code inventoryEntries}.
+     * @see InventoryDraftTransformer#transformToDraft
+     *
+     * @param inventoryEntries list of {@link InventoryEntry} from which drafts will be created
+     * @return {@link List} of {@code}
+     */
+    public static List<InventoryEntryDraft> transformToDrafts(@Nonnull final List<InventoryEntry> inventoryEntries) {
```

Comment:
Guess it is important to mention in the doc do the input and output lists items correspond to each other (e.g. have the same size and indexes). It would be enough to state that so far there is no guarantee the items are returned in the same order, but at least the sizes expected to match each other.